### PR TITLE
#159993328 Setup React-Redux

### PR DESCRIPTION
**Pull request by zakhusayn**
**What does this PR do?**
Create the appropriate folder/file structure for React-Redux workflow.

**Description of Task to be completed?**
-  Create Reducer, Action folders and appropriate file for react-redux.

#### How should this be manually tested?
-  Clone this repository and branch to `ch-redux-setup-159993328`
- Run `npm install` to install the dependencies
- Run the server `npm start -s`
- Insert random text into the input box.
- You should see the random text below it. 😃

Relevant Pivotal tracker story.
[#159993328](https://www.pivotaltracker.com/story/show/159993328)

### CODE REVIEW

**Review by hoslack — APPROVED** [review]
I approve that @blac-siren and I have worked on a basic Redux setup and is working. We welcome any changes that might be requested though.

**Comment by munyili-samuel** [comment]
@blac-siren Please give details of how this is done to make testing the PR easier
`Create Reducer, Action folders and appropriate file for react-redux`

**Comment by kisakyegordon** [comment]
@blac-siren kindly attend to the conflicts - they are being brought some earlier commits. 
cc @hoslack

**Review by munyili-samuel — APPROVED** [review]
I have tested the PR and I am able to see the random text when I type on the text field.
Good work.

**Review by RuthNjeri — CHANGES_REQUESTED** [review]
The steps in your PR are incomplete. I am getting this error 
```
./src/index.js
Module not found: Can't resolve 'redux' in '/Users/waiganjo/Documents/ah-scorpion-UI/ah-scorpion-frontend/src'
```
add `npm install` as a step.

**Comment by zakhusayn** [comment]
@RuthNjeri Thanks. I'll add it.

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,42 @@
-import React from 'react';
-import './App.css';
+import React, { Component } from "react";
```
Comment (lennykioko) [inline src/App.js]: According to Airbnb eslint standards, we should use single quotes except when writing JSX.

**Review by lennykioko — CHANGES_REQUESTED** [review]
Use single quotes except when writing JSX to ensure that we are conforming to Airbnb style guides.

**Comment by hoslack** [comment]
Hey @blac-siren , some checks are still failing. Let's fix those first.

**Comment by davenjeru** [comment]
The problem is found in `package.json` file. First of all, there are duplicate keys. Second, there is a missing comma which is bringing the error on TravisCI.

**Review by RuthNjeri — APPROVED** [review]

**Review by hoslack — CHANGES_REQUESTED** [review]
Hey @blac-siren , please remove the double quotation marks and replace with singles.

**Review by hoslack — APPROVED** [review]

**Comment by kipropbrian** [comment]
What are the relevant Pivotal Tracker Stories?

**Review by lennykioko — APPROVED** [review]
Nice work implementing the feedback. Moving forward, we should work to conform to Airbnb style guides.

**Comment by kisakyegordon** [comment]
@blac-siren kindly ensure the indenting is inorder - in particular `src/routes.test.js`

**Pull request merged**